potato_client/src/App.js

```diff
@@ -224,11 +233,124 @@ class AccommodationList extends Component {
 
 const ACCOMMODATIONS = [
   {
-    links: [{url: 'www.google.com', title: 'Google'}],
-    description: 'this is just google'
+    links: [
+      {url: "http://www.westinaustindowntown.com/?_aa_Google_My_Business__WI_3899__NAD_FM&SWAQ=958P",
+        title: "The Westin Austin Downtown:"}
+      ],
+    description: "292 feet from the venue, a one-minute walk."
+  },
+  {
+    links: [
+      {url: "http://www.marriott.com/hotels/travel/ausrd-residence-inn-austin-downtown-convention-center/?scid=bb1a189a-fec3-4d19-a255-54ba596febe2",
+        title: "Residence Inn Austin Downtown"},
+      {url: "http://www.marriott.com/hotels/travel/auscd-courtyard-austin-downtown-convention-center/?scid=bb1a189a-fec3-4d19-a255-54ba596febe2",
+        title: "Courtyard Marriott Austin Downtown"}
+      ],
+    description: "These two hotels, both by Marriott, are in the same building 0.1 miles from the venue, a three-minute walk."
+  },
+  {
+    links: [
+      {url:"http://www3.hilton.com/en/hotels/texas/hilton-austin-AUSCVHH/index.html",
+        title: "Hilton Austin"}
+    ],
+    description: "0.1 miles from the venue, a three-minute walk."
+  },
+  {
+    links: [
+      {url: "http://www.driskillhotel.com/",
+        title: "The Driskill"}
+    ],
+    description: "The Driskill, an iconic Austin experience, is 0.2 miles from the venue, a four minute walk."
+  },
+  {
+    links: [
+      {url:"https://austindowntown.place.hyatt.com/en/hotel/home.html",
+        title: "Hyatt Place Austin Downtown"}
+    ],
+    description: "0.2 miles from the venue, a five-minute walk."
+  },
+  {
+    links: [
+      {url: "https://www.omnihotels.com/hotels/austin-downtown",
+        title: "Omni Austin Hotel Downtown"}
+    ],
+    description: "0.2 miles from the venue, a six-minute walk."
+  },
+  {
+    links: [
+      {url: "http://hamptoninn3.hilton.com/en/hotels/texas/hampton-inn-and-suites-austin-downtown-convention-center-AUSDTHX/index.html",
+        title: "Hampton Inn & Suites Austin-Downtown"}
+    ],
+    description: "0.3 miles from the venue, a six-minute walk."
+  },
+  {
+    links: [
+      {url: "https://www.ihg.com/intercontinental/hotels/us/en/austin/ausha/hoteldetail?cm_mmc=GoogleMaps-_-IC-_-USA-_-AUSHA",
+        title: "Intercontinental Stephen F. Austin"}
+    ],
+    description: "0.3 miles from the venue, a six-minute walk."
+  },
+  {
+    links: [
+      {url: "http://hiltongardeninn3.hilton.com/en/hotels/texas/hilton-garden-inn-austin-downtown-convention-center-AUSGIGI/index.html",
+        title: "Hilton Garden Inn Austin Downtown"}
+    ],
+    description: "0.3 miles from the venue, a six-minute walk."
+  },
+  {
+    links: [
+      {url: "http://www.marriott.com/hotels/travel/ausjw-jw-marriott-austin/?scid=bb1a189a-fec3-4d19-a255-54ba596febe2",
+        title: "JW Marriott Austin"}
+    ],
+    description: "0.4 miles from the venue, an eight-minute walk."
   }
 ];
 
+const LOCALACCOMMODATIONS = [
```

Comment:
stylistically, it's nice to separate words in all caps variable names WITH_UNDERSCORES